bldr.env.sample

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# The endpoint, key and secret for your Minio instance (see README)
+# Change these before the first install if needed
+export MINIO_ENDPOINT=http://localhost:9000
+export MINIO_BUCKET=habitat-builder-artifact-store.local
+export MINIO_ACCESS_KEY=depot
+export MINIO_SECRET_KEY=depot
```

Comment:
The secret key in `provision.sh` below is `password`, not `depot`.  I'm guessing these should match?

Reply:
Yep, good catch!